CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Contributing to Talview Proview Moodle (Quizaccess-Proctor) Plugin
```

Comment:
@rohansharmasitoula Update this to Proview - Moodle Plugin (quizaccess_proctor)

Reply:
Updated